### DISCUSSION TITLE
Cache/history for query (calculator)

### DISCUSSION THREAD

**Discussion by AlesRFK** [Flow-Launcher/Flow.Launcher#2615] [General]
Since Launchy, I missed this feature not only for calculations, but also for other commands. I tried KP which has a problem with memory limitation/absence. Is there a similar or different problem to add this cache feature to FL?
![Rainmeter_xwaX3wsUfX](https://github.com/Flow-Launcher/Flow.Launcher/assets/13176180/fcbbf7fa-cfa3-4d61-bec6-d5ece63ab7b3)

**Comment by jonathan-f-silva** [comment]
It is possible to save the calculator history just by pressing enter to copy the result, then you can navigate to previous queries using alt+up/down. And the last thing typed always appears by default.
![Flow launcher](https://github.com/user-attachments/assets/ed04a049-8755-4706-be8a-27fe231abe0f)

**Reply by AlesRFK** [reply]
Yep, thank you, this is something that I discovered with a new release. Launchy used OutOfFocus as Enter event but I can manage :)

**Comment by jjw24** [comment]
This is also a good feature to add, I will convert to enhancement issue.
